website/docs/intro.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+---
+
+# Intro
+
+## Install
+
+Install package [`@logion/extension`](https://www.npmjs.com/package/@logion/extension) with your favorite package manager (recommended).
+Note that the extension embeds the [`@logion/client`](https://www.npmjs.com/package/@logion/client).
```

Comment:
It does not embed it strictly speaking, it brings it in as a dependency. But I guess it's OK.

Reply:
Wording adapted.